### PR TITLE
Activities menu and HSF Seminars

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
I have restructured the HSF activities / WG area so that now everything is an "activity". In addition I created a home for inactive areas that we want to archive without losing their history.

The guide for conveners is updated to reflect the new naming, but also to emphasise the options afforded by the HSF Seminar Series (the guide is linked from the menu item).

The HSF Seminar Series is also added to the "Meetings" section. As a bonus, I specifically advertised our first event on the front page and did a special logo for the seminars.

### CODE REVIEW

File: what_are_activities.md [hunk 1]
```diff
@@ -2,29 +2,48 @@
...
+
+## Archive
+
+The HSF has had a number of activities in the past which are no longer active,
+but are kept for reference:
 
 <ul class="list">
-{% for activity in site.activities %}
-  <li> <a href="{{ activity.url }}">{{ activity.title }}</a></li>
+{% for act in site.activities-archive %}
```
Comment (hegner) [inline what_are_activities.md:46]: any reason for shortening?

**Review by hegner — CHANGES_REQUESTED** [review]
Thanks a lot!
Is there a chance to put in temporary forwards as the old WG links will be rendered invalid

File: what_are_activities.md (hunk 1 above)
Comment (graeme-a-stewart) [reply what_are_activities.md]: Only laziness 😉. Fixed.

**Review by hegner — APPROVED** [review]
Thanks a lot!

**Comment by graeme-a-stewart** [comment]
> Thanks a lot! Is there a chance to put in temporary forwards as the old WG links will be rendered invalid

Good point - yes, I have done this for all the migrated pages now.

**Comment by hegner** [comment]
@graeme-a-stewart - can you deal with the merge conflicts? Thanks

**Comment by graeme-a-stewart** [comment]
> @graeme-a-stewart - can you deal with the merge conflicts? Thanks

It was already in the pipeline (my `main` hadn't been quite up to date when I started)

File: _layouts/main.html [hunk 2]
```diff
@@ -48,7 +48,7 @@ <h2>Meetings</h2>
     <p>
       The HSF holds <a href="{{ '/future-events.html' | relative_url }}" target="_hsf_indico">regular meetings</a>
       in its activity areas and has bi-weekly coordination meetings as well.
-      All of our meetings are open for everyone to join.
+      All are welcome to join these meetings and other HSF activities.
```
Comment (eduardo-rodrigues) [inline _layouts/main.html:51]:
```suggestion
      All are welcome to join these public meetings and other HSF activities.
```

File: activity-coordination.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  to provide a community forum for unsolved problems
+- should ensure there is plenty of time for discussion
+- should take notes or minutes for important discussion points (attaching these
+  to Indico is recommended)
+
+#### HSF Seminars and Cross Interest Meetings
+
+Another route for a group is that instead of group specific meetings, organise
+contributors to the [HSF Seminar Series]({{ site.baseurl
+}}/meetings/hsf-seminars.html). This meeting has a regular cadence and would
```
Comment (eduardo-rodrigues) [inline activity-coordination.md:38]:
```suggestion
}}/meetings/hsf-seminars.html). These meetings have a regular cadence and would
```

File: activity-coordination.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+- The meeting is being recorded and will be made public;
+- And where the video will be posted ([YouTube]({{ site.baseurl }}/organization/youtube.html), [CDS](http://cds.cern.ch/record/2289430), etc.);
+  - Note that it is strongly recommended not to add video files directly into Indico, instead post a link to CDS.
+
+In any case, *please also make sure that meetings are minuted in a concise way*
+(it’s not always the case that people have the time to watch the 60 minute
+video and they need a shorter summary). Minutes can be posted directly onto
+Indico (easy) or to the [HSF website]({{ site.baseurl }}/howto-website.html) (via a PR).
+
+### HSF Affiliated Projects
```
Comment (eduardo-rodrigues) [inline activity-coordination.md:67]: This is related to what we are preparing specifically. For the moment, what do you think of adding a new paragraph saying "A page dedicated to HSF Affiliated Projects and Software is under preparation, and will be linked here in due time."?

File: activity-coordination.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+We strongly encourage the use of common software in the HSF and opportunities
+for common development and sharing should be seized when they exist. The HSF
+exists as a [GitHub organisation](https://github.com/HSF) and is happy
+to host projects to help them grow. This can include things like common
+datasets used in development as well as actual code.
+
+### Coordinator Responsibilities
+
+Our working group conveners should usually dedicate between 5% and 10% of their
```
Comment (eduardo-rodrigues) [inline activity-coordination.md:77]:
```suggestion
Our activity conveners should usually dedicate between 5% and 10% of their
```

File: activity-coordination.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+time to running the working group. More than 10% is great, but we recognise
```
Comment (eduardo-rodrigues) [inline activity-coordination.md:78]:
```suggestion
time to running the activity area. More than 10% is great, but we recognise
```

File: activity-coordination.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+that people are busy with many other tasks. Less than 5% and the working group
```
Comment (eduardo-rodrigues) [inline activity-coordination.md:79]:
```suggestion
that people are busy with many other tasks. Less than 5% and the activity area
```

File: activity-coordination.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+can fall into an inactive state. Conveners should meet at least once a month to
+plan. The HSF Steering Group is always happy to attend to give feedback
+and suggestions.
+
+At least one of the working group conveners should attend the bi-weekly
```
Comment (eduardo-rodrigues) [inline activity-coordination.md:84]:
```suggestion
At least one of the activity conveners should attend the bi-weekly
```

File: activity-coordination.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+[HSF Coordination Meetings](https://indico.cern.ch/category/7970/) and
+give a short update on the group's recent and forthcoming activities.
+
+When the HSF organises [workshops](https://indico.cern.ch/category/7971/)
+working group conveners can give input to the organising team. If there
```
Comment (eduardo-rodrigues) [inline activity-coordination.md:89]:
```suggestion
activity conveners can give input to the organising team. If there
```

File: activity-coordination.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+will be a session dedicated to the activities of any working group then
```
Comment (eduardo-rodrigues) [inline activity-coordination.md:90]:
```suggestion
will be a session dedicated to matters of relevance to any activity area then
```

**Review by eduardo-rodrigues — APPROVED** [review]
Hi Graeme, this is a really nice update! I left some suggestions. HTH.

Many thanks 👍 !

File: activity-coordination.md (hunk 3 above)
Comment (hegner) [reply activity-coordination.md:38]: This I leave up to the native speaker

File: activity-coordination.md (hunk 4 above)
Comment (hegner) [reply activity-coordination.md:67]: Better we just finish and add then. Forward declarations on websites sometimes can end up being just dead wood. Which we don't want to have :-)

Comment (eduardo-rodrigues) [reply activity-coordination.md:67]: That is certainly true :).

Well, to be fair, I did my share, I'm just waiting for "final" comments and an OK to go for those docs to be shared within the Forum. Let's do just that at this coming Monday meeting.

Comment (hegner) [reply activity-coordination.md:67]: Absolutely! :-)

File: activity-coordination.md (hunk 3 above)
Comment (graeme-a-stewart) [reply activity-coordination.md]: Native speaker says yes, it's better as a plural!

**Comment by graeme-a-stewart** [comment]
Thanks for the checks and suggestions, @hegner @eduardo-rodrigues. This can be merged now, then tweaked as needed.

**Pull request merged**